ports/stm32/boards/STM32H747I_DISCO/pins.csv

```diff
@@ -0,0 +1,310 @@
+PA0,PA0
+PA1,-PA1
+PA2,-PA2
+PA3,-PA3
+PA4,-PA4
+PA5,-PA5
+PA6,-PA6
+PA7,-PA7
+PA8,PA8
+PA9,PA9
+PA10,PA10
+PA11,-PA11
+PA12,PA12
+PA13,PA13
+PA14,PA14
+PA15,PA15
+PB0,-PB0
+PB1,-PB1
+PB2,PB2
+PB3,PB3
+PB4,PB4
+PB5,-PB5
+PB6,PB6
+PB7,PB7
+PB8,PB8
+PB9,PB9
+PB10,-PB10
+PB11,-PB11
+PB12,-PB12
+PB13,-PB13
+PB14,PB14
+PB15,PB15
+PC0,-PC0
+PC1,-PC1
+PC2,PC2
+PC3,PC3
+PC4,-PC4
+PC5,-PC5
+PC6,PC6
+PC7,PC7
+PC8,PC8
+PC9,PC9
+PC10,PC10
+PC11,PC11
+PC12,PC12
+PC13,PC13
+PC14,PC14
+PC15,PC15
+PD0,PD0
+PD1,PD1
+PD2,PD2
+PD3,PD3
+PD4,PD4
+PD5,PD5
+PD6,PD6
+PD7,PD7
+PD8,PD8
+PD9,PD9
+PD10,PD10
+PD11,-PD11
+PD12,PD12
+PD13,PD13
+PD14,PD14
+PD15,PD15
+PE0,PE0
+PE1,PE1
+PE2,PE2
+PE3,PE3
+PE4,PE4
+PE5,PE5
+PE6,PE6
+PE7,PE7
+PE8,PE8
+PE9,PE9
+PE10,PE10
+PE11,PE11
+PE12,PE12
+PE13,PE13
+PE14,PE14
+PE15,PE15
+PF0,PF0
+PF1,PF1
+PF2,PF2
+PF3,PF3
+PF4,PF4
+PF5,PF5
+PF6,-PF6
+PF7,-PF7
+PF8,PF8
+PF9,-PF9
+PF10,PF10
+PF11,PF11
+PF12,PF12
+PF13,PF13
+PF14,PF14
+PF15,PF15
+PG0,PG0
+PG1,PG1
+PG2,PG2
+PG3,PG3
+PG4,PG4
+PG5,PG5
+PG6,-PG6
+PG7,PG7
+PG8,PG8
+PG9,-PG9
+PG10,PG10
+PG11,-PG11
+PG12,-PG12
+PG13,-PG13
+PG14,-PG14
+PG15,PG15
+PH0,PH0
+PH1,PH1
+PH2,-PH2
+PH3,-PH3
+PH4,-PH4
+PH5,PH5
+PH6,PH6
+PH7,PH7
+PH8,PH8
+PH9,PH9
+PH10,PH10
+PH11,PH11
+PH12,PH12
+PH13,PH13
+PH14,PH14
+PH15,PH15
+PI0,PI0
+PI1,PI1
+PI2,PI2
+PI3,PI3
+PI4,PI4
+PI5,PI5
+PI6,PI6
+PI7,PI7
+PI8,PI8
+PI9,PI9
+PI10,PI10
+PI11,-PI11
+PI12,PI12
+PI13,PI13
+PI14,PI14
+PI15,PI15
+PJ0,PJ0
+PJ1,PJ1
+PJ2,PJ2
+PJ3,PJ3
+PJ4,PJ4
+PJ5,PJ5
+PJ6,PJ6
+PJ7,PJ7
+PJ8,PJ8
+PJ9,PJ9
+PJ10,PJ10
+PJ11,PJ11
+PJ12,PJ12
+PJ13,PJ13
+PJ14,PJ14
+PJ15,PJ15
+PK0,PK0
+PK1,PK1
+PK2,PK2
+PK3,PK3
+PK4,PK4
+PK5,PK5
+PK6,PK6
+PK7,PK7
+
+# LEDs
+LED1,PI12
+LED2,PI13
+LED3,PI14
+LED4,PI15
+LED_GREEN,PI12
+LED_ORANGE,PI13
+LED_RED,PI14
+LED_BLUE,PI15
+LED_BUILTIN,PI12
+
+# Button
+SW0,PC13
+USER_BTN,PC13
+
+# UART
+UART1_TX,PA9
+UART1_RX,PA10
+UART8_TX,PJ8
+UART8_RX,PJ9
+
+# I2C
+I2C4_SCL,PD12
+I2C4_SDA,PD13
+
+# SPI
+SPI5_NSS,PK1
+SPI5_SCK,PK0
+SPI5_MISO,PJ11
+SPI5_MOSI,PJ10
+
+# USB High-Speed via ULPI
+USB_ULPI_CK,PA5
+USB_ULPI_D0,PA3
+USB_ULPI_D1,PB0
+USB_ULPI_D2,PB1
+USB_ULPI_D3,PB10
+USB_ULPI_D4,PB11
+USB_ULPI_D5,PB12
+USB_ULPI_D6,PB13
+USB_ULPI_D7,PB5
+USB_ULPI_STP,PC0
+USB_ULPI_DIR,PI11
+USB_ULPI_NXT,PH4
```

Comment:
Do the USB pins need to be exposed with names?  All the peripherals below are not exposed to the user (ETH, SDMMC, FMC).